server/sched.go

```diff
@@ -537,7 +537,7 @@ func pickBestFitGPUs(req *LlmRequest, ggml *llm.GGML, gpus gpu.GpuInfoList) gpu.
 }
 
 // findRunnerToUnload finds a runner to unload to make room for a new model
-func (s *Scheduler) findRunnerToUnload(req *LlmRequest) *runnerRef {
+func (s *Scheduler) findRunnerToUnload() *runnerRef {
```

Comment:
this is just fixing an unused argument linter error

Reply:
I had intended to use the new request to pick which runner to unload if there were multiples, but haven't implemented that yet, so this is fine to remove and we can add it back when/if we implement smarter unload selection.

Reply:
Ah, I had figured. Ok! Will keep it out to please the linter gods for now 😊